### PR TITLE
support invocationDetails of cypress

### PR DESCRIPTION
**Pull request by Franckrst**
I show we can have all information for unit test report in invocationDetails of suite.
So we can use it ?

### CODE REVIEW

File: src/ReporterUtils.js [hunk 1]
```diff
@@ -155,5 +165,6 @@ module.exports = {
     formatTest,
     formatTestTitle,
     formatSuiteTitle,
-    writeFile
+    writeFile,
+    hasSpecInTitle
```
Comment (BBE78) [inline src/ReporterUtils.js:169]: Provide a test to this new exported function (see test\specs\ReporterUtils.spec.js)

File: src/SonarQubeCypressReporter.js [hunk 2]
```diff
@@ -66,14 +67,28 @@ class SonarQubeCypressReporter {
...
-            node.attribute("path", specFilePath);
+
+            if (hasSpecInTitle(suite)) {
+
+                this.specFilename = extractSpecFromSuite(suite, {useAbsoluteSpecPath: false});
+                const specFilePath = extractSpecFromSuite(suite, this.options);
+                suite.title = extractTitleFromSuite(suite);
+                node.attribute("path", specFilePath);
+
+            } else {
```
Comment (BBE78) [inline src/SonarQubeCypressReporter.js:79]: Provide a test case that demonstrate this branch

**Review by BBE78 — CHANGES_REQUESTED** [review]
Please update the `README` to describe the new feature: from what I understood, Mocha provide an `invocationDetails` property that could contain the same information and make the `specTitle` useless.
I could not find any description about the `invocationDetails` property on Mocha: could you please provide a link to its documentation and products version

**Comment by Franckrst** [comment]
> I could not find any description about the `invocationDetails` property on Mocha: could you please provide a link to its documentation and products version

I don't show documentation about `invocationDetails` but we can see it in cypress code : https://github.com/cypress-io/cypress/search?o=asc&q=invocationDetails&s=indexed

**Comment by Franckrst** [comment]
i make all change you want. But for the readme i have very bad english, ca you make modification ?

**Comment by Franckrst** [comment]
i dont understand why travis fail. I have no fail on my machine

**Comment by BBE78** [comment]
> i dont understand why travis fail. I have no fail on my machine

Troubles with Yarn cache and Cypress...

**Comment by BBE78** [comment]
I have some troubles with your PR: the test did not work on my env.
image

**Comment by BBE78** [comment]
@Franckrst without any news from your side, I'll reject this PR...

**Comment by BBE78** [comment]
No news from author @Franckrst since too many months, rejecting this PR

**Pull request closed without merging**